src/appleseed.studio/mainwindow/mainwindow.ui

```diff
@@ -404,6 +405,14 @@
     <string>F5</string>
    </property>
   </action>
+  <action name="action_rendering_pause_rendering">
+   <property name="text">
+    <string>Pause &amp;Rendering</string>
```

Comment:
That conflicts with `&Rendering Settings`. I suggest putting the & on the P of Pause.

Reply:
by the way, what's the purpose of &amp; ?

Reply:
It creates a keyboard shortcut: the character following the & gets underlined:

![image](https://user-images.githubusercontent.com/321290/41848351-777456ba-787d-11e8-8fb3-5731ed4311a1.png)

Reply:
That way you can do Alt+F followed by N to create a new project.